tools/cassandra/setupTask.go

```diff
@@ -33,7 +33,8 @@ type SetupSchemaTask struct {
 }
 
 func newSetupSchemaTask(config *SetupSchemaConfig) (*SetupSchemaTask, error) {
-	client, err := newCQLClient(config.CassHosts, config.CassKeyspace)
+	client, err := newCQLClient(config.CassHosts, config.CassPort, config.CassUser, config.CassPassword,
+		config.CassKeyspace)
```

Comment:
yeah, 120 chars is the correct config, there was an additional newline.. May be its just the diff viewer, ignore.